README.rst

```diff
@@ -0,0 +1,76 @@
+Example: Basic Sphinx project for Read the Docs
+===============================================
+
+This example shows the an integration of a basic Sphinx project with Read the Docs. You're encouraged to view it to get inspiration and copy & paste from the files in the source code. If you are using Read the Docs for the first time, have a look at the official `Read the Docs Tutorial <https://docs.readthedocs.io/en/stable/tutorial/index.html>`_.
+
+📚 `docs/ <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/docs/>`_
+    A basic Sphinx project lives in ``docs/``, it was generated using Sphinx defaults
+⚙️ `.readthedocs.yaml <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/.readthedocs.yaml>`_
+    Read the Docs Build configuration is stored in ``.readthedocs.yaml`` (configuration-as-code)
+📍 `docs/requirements.txt <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/docs/requirements.txt>`_ and `docs/requirements.in <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/docs/requirements.in>`_
+    Python dependencies are `pinned <https://docs.readthedocs.io/en/latest/guides/reproducible-builds.html>`_ (uses `pip-tools <https://pip-tools.readthedocs.io/en/latest/>`_)
+💡 `docs/api.rst <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/docs/api.rst>`_
+    By listing the Python module ``lumache`` in ``autosummary``, Sphinx will automatically scan this module and generate API docs.
```

Comment:
Some of these list items end with a period, and some don't. We should standardize on one approach, probably ending w/ the period, because otherwise multiple sentences are weird.

Reply:
Yes, I think that ending with a period is nicest. Not having a full paragraph/sentence structure for each item requires some other kind of grammar hack... which usually turns out to be... hackish.

Like what if the following list items...

* ...started like this or...
* ...continued to mention alternatives without having full sentences
* ...were all this annoying to read!

Often I start a list like the above - during the writing process - and then end up turning it into full and proper sentences :)